packages/web/src/scss/components/TextArea/README.md

```diff
@@ -71,10 +71,29 @@ modifier to fill the available space.
 ## Validation States
 
 Validation states can be presented either by adding a CSS modifier class
-(`TextArea--success`, `TextArea--warning`, `TextArea--error`), or by adding
+(`TextArea--success`, `TextArea--warning`, `TextArea--danger`, `TextArea--error`), or by adding
```

Comment:
Again, please link this to the docs/dictionaries.md

Reply:
done